week10/threeway-merge.md

```
@@ -1,5 +1,12 @@
 3-way merge는 서로 다른 두 브랜치에서 두 브랜치 모두 다른 변경사항이 생기는 경우 사용되는 merge 방식입니다.
+<<<<<<< HEAD
 base가 되는 커밋을 기준으로 각 브랜치의 변경 내용을 비교하여 병합합니다.
 이 방식은 충돌이 발생할 가능성이 있고 충돌이 발생하면 수동으로 해결해야 해요.
 
 Git의 merge 알고리즘은 base 커밋, HEAD, merge 하는 대상 브랜치의 변경 내용을 기반으로 동작합니다.
```

markdownlint-cli2 (0.17.2)

Trailing punctuation in heading
Punctuation: '.'

(MD026, no-trailing-punctuation)